pkgs/applications/graphics/veusz/default.nix

```diff
@@ -39,7 +39,7 @@ python3Packages.buildPythonApplication rec {
   setupPyBuildFlags = [
     # --sip-dir does nothing here, but it should be the correct way to set the
     # sip_dir, so I'm leaving it here for future versions
-    "--sip-dir=${python3Packages.pyqt5}/share/sip"
+    "--sip-dir=${python3Packages.pyqt5}/${python3Packages.python.sitePackages}/PyQt5/bindings"
```

Comment:
```suggestion
```

Previously, I used the `substituteInPlace` because the build script ignored the `--sip-dir` flag. But now, the `--sip-dir` flag is not ignored, but we still need to override it because it does not end in `PyQt5`. I would be in favor of removing this flag, since the build works without it.

Reply:
I had quickly fixed the build without paying much attention to some details. I updated the comment explaining the new issues that came after building pyqt5 with sip 6.